agent.go

```diff
@@ -296,6 +307,9 @@ func (agent *Agent) genAgentComponentConfigsLocked() *agentComponentConfigs {
 		for _, host := range bootstrapHosts.NonSSL.Search {
 			searchEndpoints = append(searchEndpoints, "http://"+host)
 		}
+		for _, host := range bootstrapHosts.NonSSL.Analytics {
```

Comment:
In that case, we would need to keep temporaries for each of the host types and additionally depends on the fact that those 4 services all share the same protocol.  It seemed better to me to just put them individual since it makes it clear that they all happen to share the same protocol, rather than it being an actual codified rule.